app/views/candidate_interface/start_page/show.html.erb

```diff
@@ -23,6 +24,10 @@
       <li>grade 4 (C) or above in English and maths GCSEs</li>
       <li>grade 4 (C) or above in GCSE science if you want to teach primary</li>
     </ul>
+    <p>Before you start a teacher training course you’ll also need to confirm you have the health and physical capacity to start training.</p>
+
+    <h2 class="govuk-heading-m">If you don’t have a degree</h2>
+    <p class="govuk-body">You can teach in further education (age 14 to adult), or study for a degree leading to qualified teacher status. <a href="https://getintoteaching.education.gov.uk/eligibility-for-teacher-training">Learn more</a>.</p>
```

Comment:
Isn't this content already written further down? 👀

Reply:
good catch👌  I'm sure this was introduced with the rebase, I'll fix it!